### PR TITLE
[CLocal-Azure] Write unit tests cases for the project 

### PR DESCRIPTION
**Pull request by kunalbhatia18**

### CODE REVIEW

File: src/services/azure-api-app-service/azure-api-app-service.js [hunk 1]
```diff
@@ -8,13 +8,24 @@ let path = require("path");
...
 let initFile;
 let folder;
-// let workingDir = "../../example/azure-api-service/"
 
 if (process.argv[2] == "api-start") {
   folder = process.argv[3];
   initFile = process.argv[4];
 }
 
+if (navigator.appVersion.indexOf("Linux") != -1) {
```
Comment (rajikaimal) [inline src/services/azure-api-app-service/azure-api-app-service.js]: What is the relationship of this addition with the unit test? Your PR is related to unit tests I assume

File: function_test/wrapper.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"use strict";
```
Comment (rajikaimal) [inline function_test/wrapper.js]: Unitest directory should be named `test`. Can you explain are you testing with the following test? I don't see any unit test here. eg: assertion.

File: src/services/azure-functions/azure-function.js [hunk 3]
```diff
@@ -154,5 +165,13 @@ function removeContainer() {
     });
   });
 }
+import sleeps from "./sleeps.js";
+ 
+test("Request has required parameters",
```
Comment (rajikaimal) [inline src/services/azure-functions/azure-function.js]: Why did you add a unit test to this file? Besides, do you think there is any meaning to this test? If so can you please explain?

File: src/services/azure-api-app-service/azure-api-app-service.js (hunk 1 above)
Comment (kunalbhatia18) [reply src/services/azure-api-app-service/azure-api-app-service.js]: Yes they are

Comment (kunalbhatia18) [reply src/services/azure-api-app-service/azure-api-app-service.js]: this is for adding windows and linux support to clocal-azure project

**Comment by PasinduPriyashan** [comment]
This PR contains changes for two different tasks. Make PRs for each one of them individually.
